server/server.go

```diff
@@ -466,8 +468,12 @@ func cutPluginCommandF(w http.ResponseWriter, slashCommand *MMSlashCommand, tag,
 		return nil
 	}
 
+	releasePrefix := ""
+	if preRelease {
+		releasePrefix = "pre-"
+	}
 	command := slashCommand.Command + " " + slashCommand.Text
-	msg := fmt.Sprintf("@%s triggered a plugin release process using `%s`.\nTag %s created in`%s`. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: ", slashCommand.Username, command, tag, repo)
+	msg := fmt.Sprintf("@%s triggered a plugin %srelease process using `%s`.\nTag %s created in`%s`. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: ", slashCommand.Username, releasePrefix, command, tag, repo)
 	if err := createTag(ctx, client, Cfg.GithubOrg, repo, tag, commitSHA); errors.Is(err, ErrTagExists) {
```

Comment:
Is it possible to set the `pre-release` bit from within `createTag`, so that there's no window between it being marked as a release version and when we later set it back to `pre-release`?

Reply:
Since all are different api calls to github, there will always be some window in between it being marked as release version and when we later set it to `pre-release`. Since all apis are called serially/sequentially, so the window is directly proportional to the number of api calls in between those two lines of code.
But it still makes sense to move setting of the tag as `pre-release` between creating the tag and creating the ref from it, so that it is ensured that it will always run, or else we may still have a tag with a ref but no `pre-release` and no uploads.